README.md

```diff
@@ -1,24 +1,117 @@
-# README
+# Expense Tracker
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [ Video Description Link](#video-description)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Expense Tracker <a name="about-project"></a>
 
-* System dependencies
+**Expense Tracker** is an app which lets the user keep track and details of their expenses.
+## 🛠 Built With <a name="built-with"></a>
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
+<details>
+<summary>Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Database creation
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Database initialization
+### Key Features <a name="key-features"></a>
 
-* How to run the test suite
+This app contains
+- **Expenses and their Categories**
 
-* Services (job queues, cache servers, search engines, etc.)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
+## Video Description <a name="video-description"></a>
 
-* ...
+- [Video Description Link](https://www.loom.com/share/d5fc732c64b848e0b89c77ebdf2fe537)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Expense Tracker](rails-xez4.onrender.com)
```

Comment:
- [ ] This link leads to a 404 page when clicked. Please fix it.
![image](https://github.com/sameedmohsin28/Expense-Tracker/assets/45460192/65570dad-7715-43a6-b31c-ed38d632c758)